### PR TITLE
Expose build_runner dependencies without prefix

### PR DESCRIPTION
**Pull request by Schwusch**
build_runner doesn't handle import prefixes, show these specifically without prefix

### CODE REVIEW

**Comment by Schwusch** [comment]
This issue surfaced when I was looking at DynamoDB

File: generator/lib/library_builder.dart [hunk 1]
```diff
@@ -46,6 +46,7 @@ import 'package:json_annotation/json_annotation.dart';
 import 'package:meta/meta.dart' as _meta;
 
 import 'package:aws_client/shared.dart' as shared;
+import 'package:aws_client/shared.dart' show Uint8ListConverter, Uint8ListListConverter;
```
Comment (isoos) [inline generator/lib/library_builder.dart]: Something does not add app: either you'd need to remove the `shared.` prefix from `shared.Uint8ListConverter`, or we are missing the prefix from unprefixed `Uint8ListConverter`. (I'll need to check the head to be sure). What am I missing?

Comment (Schwusch) [reply generator/lib/library_builder.dart]: We could remove the `shared.` prefix, but I think you mentioned a clash of naming in some services? Unfortunately `build_runner` does not take prefixes into account when annotating fields like we do:
```dart
@shared.Uint8ListConverter()
@JsonKey(name: 'B')
final Uint8List b;
```
in the `xxx.g.dart` it becomes:
```dart
writeNotNull('B', const Uint8ListConverter().toJson(instance.b));
```

Comment (Schwusch) [reply generator/lib/library_builder.dart]: But you're right, it would be more consistent to remove the prefix when annotating

**Pull request merged**